### PR TITLE
 DP-22215 - SPIKE make proof of concept for location listing view

### PR DESCRIPTION
**Pull request by ddiestra**
**Description:**
Add a new view for locations to replace the current locations page and also use leaflet map

**Jira:** (Skip unless you are MA staff)
https://jira.mass.gov/browse/DP-22215

**To Test:**
Go to any  organization, service, or location page and go to /locations.

Examples: 
- orgs/massachusetts-registry-of-motor-vehicles/locations
- locations/lowell-court-service-center/locations
- court-service-centers/locations

**Screenshots/GIFs:**
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

**Comment by clairesunstudio** [comment]
@ddiestra  please update this branch and the associated mayflower package and deploy this to a feature environment

File: composer.json [hunk 1]
```diff
@@ -106,9 +106,37 @@
                     "type": "zip"
                 }
             }
+        },
```
Comment (clairesunstudio) [inline composer.json]: are these packages added intentionally?

Comment (clairesunstudio) [reply composer.json]: ok these are added for handling ajax change --> 8a380f1

**Comment by clairesunstudio** [comment]
Associated MF PR https://github.com/massgov/mayflower/pull/1430/files

**Comment by clairesunstudio** [comment]
This branch is deployed to STAGE. But the location listing and the map are not rendered. 

Screen Shot 2021-10-14 at 3 04 16 PM

File: docroot/modules/custom/mass_map/mass_map.routing.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 mass_map.map_page:
-  path: '/node/{node}/locations'
+  path: '/node/{node}/locations-old'
```
Comment (tom-fleming) [inline docroot/modules/custom/mass_map/mass_map.routing.yml:2]: @joeg8612 Is this something we want to hang onto? This is the old map, which will still be accessible at locations-old. I would prefer that we just get rid of it.

**Pull request merged**